f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -1290,6 +1307,7 @@ def _update_member_status(self, members):
                             plugin_const.ACTIVE,
                             lb_const.ONLINE
                         )
+                        member['provisioning_status'] = plugin_const.ACTIVE
```

Comment:
This will not get called in the case where a member segment is not complete because we caught the network not ready exception from prep_service_networking and as a result added the set loadbalancer_update flag to false.